flang/lib/Optimizer/OpenMP/MapInfoFinalization.cpp

```diff
@@ -247,16 +256,26 @@ class MapInfoFinalizationPass
                               mlir::omp::TargetUpdateOp>(target))
       return mapTypeFlag;
 
-    bool hasImplicitMap =
-        (llvm::omp::OpenMPOffloadMappingFlags(mapTypeFlag) &
-         llvm::omp::OpenMPOffloadMappingFlags::OMP_MAP_IMPLICIT) ==
+    llvm::omp::OpenMPOffloadMappingFlags Implicit =
```

Comment:
should likely be "implicit", with a small i, to align with the case style of everything!